src/core/timer.c

```diff
@@ -394,6 +394,15 @@ static void timer_enter_waiting(Timer *t, bool time_change) {
                 if (v->base == TIMER_CALENDAR) {
                         usec_t b, rebased;
 
+                        /* Update last_trigger to 'now' in case
+                         * the system time changes, so that
+                         * next_elapse is not stuck with a
+                         * future date. */
```

Comment:
Nowadays, we allow ~109ch per line. Please do not wrap lines so frequently.

Reply:
Changed